config.siyuan.yaml

```diff
@@ -979,3 +979,68 @@ repos:
     target: https://repo.huaweicloud.com/ubuntu-cdimage/
     disabled: true
     <<: *oneshot_common
+  # mathlib
+  - type: shell_script
+    script: /worker-script/git.sh
+    interval: 3600
+    name: git/lean4-packages/std4
+    source: https://github.com/leanprover/std4.git
+    path: /srv/data32T/git/lean4-packages/std4
+    serve_mode: git
+    target: https://git.sjtu.edu.cn/sjtug/lean4-packages/std4.git
```

Comment:
`git.sjtu.edu.cn` is another server hosted by NIC. We don't need it anymore, so you can just remove this line.